server/rpc.go

```diff
@@ -82,10 +82,11 @@ func CheckTranscoderAvailability(orch Orchestrator) bool {
 
 	ping := crypto.Keccak256(ts_signature)
 
-	orch_client, err := startOrchestratorClient(orch.ServiceURI().String())
+	orch_client, conn, err := startOrchestratorClient(orch.ServiceURI().String())
 	if err != nil {
 		return false
 	}
+	defer conn.Close()
```

Comment:
I see. We want the orchestrator connection to close after this function exits .... i.e. once we've checked that the orchestrator ("transcoder") is available. Noted!